tools/handle_rhv_storage_duplication.rb

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env ruby
+
+# This script removes duplicate block storages that were created due to changing the way
+# storage "location" is calculated.
+# It copies the tags from the old storages and then removes them from ManageIQ.
+# See https://github.com/ManageIQ/manageiq-providers-ovirt/pull/387
+# And https://bugzilla.redhat.com/show_bug.cgi?id=1697467
+# Recommended usage: run with -dv argument to have a dry run with output.
+# If it makes sense, run with -v
+
+class HandleStorageDuplication
+  attr_reader :ems, :dry_run, :verbose
+  def initialize(opts)
+    @ems = opts[:ext_management_system]
+    @dry_run = opts[:dry_run]
+    @verbose = opts[:verbose]
+  end
+
+  def effected_storages
+    @effected_storages ||= @ems ? @ems.storages.where.not(:store_type => ["NFS", "GLANCE"]) : Storage.where.not(:store_type => ["NFS", "GLANCE"])
```

Comment:
> Storage.where.not(:store_type => ["NFS", "GLANCE"])

Won't this also include e.g. VMware and SCVMM datastores?

Reply:
Yes it will, but they will not be found in any RHV provider so nothing will happen with them.
https://github.com/ManageIQ/manageiq-providers-ovirt/pull/441/files#diff-1b95d354517465f92f23cd9a063b1f8dR39

Is there a better way you would suggest?

Reply:
We could either take a particular provider id/name, or go through all Ovirt providers to get a list of storages in use by Ovirt

Reply:
@agrare I have finally proven to myself with specs why we need to go over all of them, in Ivanchuk, before your wounderful PR for master :) the storages are not always associated with a provider, they are associated through a host - this includes the case when we get storages that are down from rhv.
In that case they are not connected to any host and I do not know how to check if they are rhv storages (perhaps use some ems_ref heuristics, but we will still need to go over all storages to do that) so we have no information on which rhv they belong to.
I have the script that runs only over connected storages ready if you will still think it is better, but I think it will case a bug in some cases.

Reply:
Okay this works for me then

Reply:
I added an improvement based on your comment:
https://github.com/ManageIQ/manageiq-providers-ovirt/pull/441/files#diff-1b95d354517465f92f23cd9a063b1f8dR25

If this is ok, @agrare @gekorob can you merge when specs are green?

Reply:
@agrare this script makes sense only for ivanchuk, after the changes you made it makes no sense, should I add a PR to remove it after this one is back ported?